### DISCUSSION TITLE
Compatibility Issue with spatie/laravel-sitemap and intervention/image-laravel in Laravel 10 Project

### DISCUSSION THREAD

**Discussion by Rava-milad** [spatie/laravel-sitemap#551] [Q&A]
Hello,

I'm encountering an issue when trying to install the spatie/laravel-sitemap package in my Laravel 10 project. The problem arises specifically when the intervention/image-laravel package is also included.

Here are the details:

Laravel Version: 10.10
PHP Version: ^8.1
Packages Installed:
artesaos/seotools: ^1.2
guzzlehttp/guzzle: ^7.2
intervention/image-laravel: *
laravel/framework: ^10.10
laravel/sanctum: ^3.3
laravel/tinker: ^2.8
league/flysystem-aws-s3-v3: ^3.22
morilog/jalali: 3.*
spatie/laravel-sitemap: *
Issue Description:

When intervention/image-laravel is included in the project, the installation of spatie/laravel-sitemap fails. The issue seems to be resolved when intervention/image-laravel is removed.

Steps to Reproduce:

Install the Laravel 10 project.
Add the intervention/image-laravel package.
Attempt to install spatie/laravel-sitemap.
Observe that the installation fails.
Expected Behavior:

spatie/laravel-sitemap should install successfully and be compatible with intervention/image-laravel in a Laravel 10 project.

Actual Behavior:

The installation of spatie/laravel-sitemap fails when intervention/image-laravel is present.

Additional Information:

Please let me know if there are any known compatibility issues or if there are specific version constraints for these packages.
Any guidance or suggestions to resolve this conflict would be greatly appreciated.
Thank you!

**Comment by anburocky3** [comment]
Any update on this?

**Reply by Rava-milad** [reply]
Not for these settings

**Comment by xplicit-dev** [comment]
removed "spatie/laravel-sitemap" package from my project and used watson/sitemap

**Comment by Rava-milad** [comment]
Ultimately, I was forced to upgrade the PHP version of the project to 8.2 in order to use this package.
